README.rst

```diff
@@ -138,6 +138,44 @@ The set of repositories to operate on can optionally be restricted by the type:
 If the command should work on multiple repositories make sure to pass only generic arguments which work for all of these repository types.
 
 
+Access repositories that require authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Vcstool supports authenticated access to repositories by parsing files with a netrc-like format.
```

Comment:
I think this should mention that this is only relevant for the `tar` and `zip` types (assuming my assumption is right).

Reply:
Your assumption is correct, rephrased.